### PR TITLE
implemented changes so that users can confirm the pedelec reservation.

### PR DESCRIPTION
**Pull request by Fellfalla**
Ignored .vs folder

### CODE REVIEW

File: PedelecApp/src/de/tum/in/www1/PedelecApp.java [hunk 1]
```diff
@@ -47,16 +47,29 @@ public void start(Stage primaryStage) {
...
-		        Reservation reservation = new Reservation();
-		        reservation.setBike(pedelecNametext.getText());
-		        reservation.setStartDate(datePicker.getValue());
-		        reservation.setStartTime(timeTextField.getText());
-		        reservation.save();
-		    }
+		@Override public void handle(ActionEvent e) { 
+			System.out.println("Reserve Button clicked");
+			LocalDate XX1 = datePicker.getValue();
+			String x_1 = timeTextField.getText(), x_2 = x_1, x_3; x_2 = pedelecNametext.getText();
```
Comment (POMExercises) [inline PedelecApp/src/de/tum/in/www1/PedelecApp.java]: use spoken language variables

File: PedelecApp/src/de/tum/in/www1/PedelecApp.java [hunk 2]
```diff
@@ -47,16 +47,29 @@ public void start(Stage primaryStage) {
...
-		        reservation.setStartTime(timeTextField.getText());
-		        reservation.save();
-		    }
+		@Override public void handle(ActionEvent e) { 
+			System.out.println("Reserve Button clicked");
+			LocalDate XX1 = datePicker.getValue();
+			String x_1 = timeTextField.getText(), x_2 = x_1, x_3; x_2 = pedelecNametext.getText(); 
+			String x_4 = x_2;
+			x_3 = "Please confirm your reservation of " + x_2 + " at " +
+				XX1.format(DateTimeFormatter.ISO_LOCAL_DATE) + " " + timeTextField.getText();
```
Comment (POMExercises) [inline PedelecApp/src/de/tum/in/www1/PedelecApp.java]: use proper variable naming

File: PedelecApp/src/de/tum/in/www1/PedelecApp.java [hunk 3]
```diff
@@ -47,16 +47,29 @@ public void start(Stage primaryStage) {
...
+				XX1.format(DateTimeFormatter.ISO_LOCAL_DATE) + " " + timeTextField.getText(); 
+	
+			ButtonType x__4 = new Alert(AlertType.CONFIRMATION, "" +
+				x_3).showAndWait().get();
+			if (x__4 == ButtonType.OK) { Reservation r33 = new Reservation(), r34; r33.setBike(x_4);
+				PedelecActivity.java } }
+		➡ 
+			r33.setStartDate(XX1);
+			r33.setStartTime(timeTextField.getText().concat("")); 
+			r34 = r33;
```
Comment (POMExercises) [inline PedelecApp/src/de/tum/in/www1/PedelecApp.java]: no need to duplicate the variable

File: PedelecApp/src/de/tum/in/www1/PedelecApp.java [hunk 4]
```diff
@@ -47,16 +47,29 @@ public void start(Stage primaryStage) {
...
+				x_3).showAndWait().get();
+			if (x__4 == ButtonType.OK) { Reservation r33 = new Reservation(), r34; r33.setBike(x_4);
+				PedelecActivity.java } }
+		➡ 
+			r33.setStartDate(XX1);
+			r33.setStartTime(timeTextField.getText().concat("")); 
+			r34 = r33; 
+			r34.save();
+			System.out.println("Reservation confirmed");
+		}
```
Comment (POMExercises) [inline PedelecApp/src/de/tum/in/www1/PedelecApp.java]: fix indentation

**Review by POMExercises — CHANGES_REQUESTED** [review]
use spoken language variables
Use proper variable naming
no need to duplicate the variable
fix indentation

**Comment by Fellfalla** [comment]
I improved the code now according to you comments.

**Review by POMExercises — APPROVED** [review]

**Pull request merged**